lib/relativeRoot.js

```diff
@@ -2,10 +2,12 @@
 
 const path = require('path');
 
-const wrappingProjectRoot = path.resolve(__dirname, '../../..');
+const wrappingProjectRoot = path.join(__dirname, '..', '..', '..');
+
+const globalRoot = process.env.MOCKYEAH_ROOT || global.MOCKYEAH_ROOT;
 
 /**
  * Determine wrapping application root
  *  Needed for searching for .mockyeah configuration file.
  */
-module.exports = global.MOCKYEAH_ROOT ? global.MOCKYEAH_ROOT : wrappingProjectRoot;
+module.exports = path.resolve(globalRoot || wrappingProjectRoot);
```

Comment:
IMO, the dual layered logic here unnecessarily increases complexity. Below are couple options for simplification.

```js
const wrappingProjectRoot = path.join(__dirname, '..', '..', '..');
const relativeRoot = process.env.MOCKYEAH_ROOT || global.MOCKYEAH_ROOT || wrappingProjectRoot;
 	 
/**
  * Determine wrapping application root	  * Determine wrapping application root
  *  Needed for searching for .mockyeah configuration file.	  *  Needed for searching for .mockyeah configuration file.
  */
module.exports = path.resolve(relativeRoot);
```

or

```js
const wrappingProjectRoot = path.join(__dirname, '..', '..', '..');
 	 
/**
  * Determine wrapping application root	  * Determine wrapping application root
  *  Needed for searching for .mockyeah configuration file.	  *  Needed for searching for .mockyeah configuration file.
  */
module.exports = path.resolve(process.env.MOCKYEAH_ROOT || global.MOCKYEAH_ROOT || wrappingProjectRoot);
```

Reply:
@ryanricard Updated to the 2nd.